### ISSUE TITLE
Add subtitle and date fields to posts

### ISSUE THREAD

**Issue by kirsten** [luncheonette/the-sunview#15]
Some events have subtitles/subheaders on the current site ([example](http://thesunview.org/roman-osminkin-not-word-politics/)). We should preserve this feature.

**Comment by kirsten** [comment]
@dgoats I broke this out into its own issue. I've got the subtitles up already (see example [here](https://github.com/luncheonette/the-sunview/blob/master/_posts/2016-04-07-osminkin.md)) - should also be working in the CMS.

I can set up the date field a couple of ways:

1. Add a field for only event time, and grab the event date from the post itself (since it already includes that in the filename). So you'd only have to enter "7 PM Doors | 730 PM Reading" in an event time field, and the page would automatically display "Thursday April 07, 2016 @ 7 PM Doors | 730 PM Reading".
2. Add a custom date field where you'd enter the whole "Thursday April 07, 2016 @ 7 PM Doors | 730 PM Reading". Might be good to have this flexibility if we anticipate having events that span multiple days or something...

1 is less typing, standardized date formatting, but 2 is super flexible. Either is easy! Thoughts?
